.github/workflows/pr-checklist.yaml

```diff
@@ -28,6 +28,9 @@ on:
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+permissions:
```

Comment:
can you add a comment explaining why we need that, otherwise in 6 months we will have forgotten and most people in the team are not devops/GHA expert, so better to be very verbose with comment here.

Reply:
Done! I'll wait for CI to pass, then will merge.